packages/components/package.json

```diff
@@ -38,17 +38,23 @@
     },
     "peerDependencies": {
         "react": "*",
-        "react-dom": "*"
+        "react-dom": "*",
+        "@hopper-ui/styled-system": "*",
+        "react-aria-components": "^1.1.0",
+        "react-aria": "^3.32.0"
     },
     "dependencies": {
-        "@hopper-ui/styled-system": "workspace:*"
+        "@hopper-ui/icons": "workspace:*",
```

Comment:
Component will have a dependency on icons, since a couple of component will use icons (search input, close icon, arrows).  At the moment, i only need it for storybook tests. i could move it to dev dependency for now and promote it later